src/main/scala/wdl4s/expression/FileEvaluator.scala

```diff
@@ -115,7 +115,7 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
       case a: Ast if a.isArrayOrMapLookup =>
         evalValue(a) match {
           case Success(f: WdlFile) => Success(Seq(f))
-          case _ => evaluateRecursive(a.getAttribute("rhs"))
+          case _ => evaluateRecursive(a.getAttribute("lhs"))
```

Comment:
If there's an expression on the rhs that's no longer going to be evaluated?  Why don't we want to evaluate both sides?